src/services/api/index.ts

```diff
@@ -439,10 +439,10 @@ export enum METRIC_TYPES {
 
 export async function saveMetric({event = ''}) {
   try {
-    const analyticsOptin = await SecureStore.getItemAsync(
+    const analyticsConsent = await SecureStore.getItemAsync(
```

Comment:
I know you've used `analyticsConsent` but it feels to me we're normally referring to it as `analyticsOptIn`. Also not to be used as the short term "consent" and to be mixed with "check-in consent". Up to you in the end

Reply:
Yeah I switched it to `consent` partly because it felt weird to write `...OptIn = true` but it makes more sense to match the RNENS term especially now that part is explained.